lib/navigation.dart

```diff
@@ -72,6 +84,7 @@ class BackdropNavigationBackLayer extends StatelessWidget {
     return ListView.separated(
       shrinkWrap: true,
       itemCount: items.length,
+      padding: itemPadding,
       itemBuilder: (context, position) => InkWell(
         child: items[position],
```

Comment:
```suggestion
        child: Container(
          decoration: ShapeDecoration(shape: itemCustomBorder),
          child: items[position],
        ),
```

Reply:
Oh I see. But what does this exactly change compared to directly using `customBorder` attribute of `InkWell`?

Reply:
I am saying do both, set it as `customBorder` for `InkWell` and add `ShapeDecoration` around item, so the shape is consistent, since we are naming it `itemShape`(or `itemCustomBorder` as of now).

If we just want to use it with `InkWell` we should change the name to `itemSplashShape` or `itemSplashBorder`.

Reply:
Oh now I get it ;)